<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_wind</class>
 <widget class="QDialog" name="Login_wind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>31</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Авторизация</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_line">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="pasword_line">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>21</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_new_acc_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Новый аккаунт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
